data/map planets.txt

```diff
@@ -3631,6 +3658,13 @@ planet Quirinal
 	government Uninhabited
 	security 0
 
+planet "Rabasot Pa'ah"
+	attributes ocean uninhabited fishing korath "korath exiles"
+	landscape land/rough-seas
+	description `This tectonically active ocean world has unpredictable weather, frequent tsunamis, and unusual electrical activity. Although there are a few islands, the terrain is so harsh that it is nearly impossible to land and the atmosphere is bathed in radiation from Kor Ak'Mari's death throes. There's almost no vegetation in sight, but the sea is filled with microscopic lifeforms. It seems the violent conditions on this planet have held back evolution.`
```

Comment:
```suggestion
	description `This tectonically active ocean world has unpredictable weather, frequent tsunamis, and unusual electrical activity. Although there are a few islands, the terrain is so harsh that it is nearly impossible to land, and the atmosphere is bathed in radiation from Kor Ak'Mari's death throes. There is almost no vegetation in sight, but the sea is filled with microscopic lifeforms. It seems the violent conditions on this planet have held back evolution.`
```

Reply:
I am going to trust that this change will not cause overflow..